{
	"schema_version": "1.6.0",
	"id": "UBUNTU-CVE-2018-7753",
	"modified": "2024-10-15T14:06:43Z",
	"published": "2018-03-07T23:29:00Z",
	"related": [
		"CVE-2018-7753"
	],
	"details": "An issue was discovered in Bleach 2.1.x before 2.1.3. Attributes that have URI values weren't properly sanitized if the values contained character entities. Using character entities, it was possible to construct a URI value with a scheme that was not allowed that would slide through unsanitized.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:Pro:16.04:LTS",
				"name": "python-bleach",
				"purl": "pkg:deb/ubuntu/python-bleach?arch=src?distro=esm-apps/xenial"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						}
					]
				}
			],
			"versions": [
				"1.4-1",
				"1.4.2-1"
			],
			"ecosystem_specific": {
				"ubuntu_priority": "medium"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2018/UBUNTU-CVE-2018-7753.json"
			}
		},
		{
			"package": {
				"ecosystem": "Ubuntu:Pro:18.04:LTS",
				"name": "python-bleach",
				"purl": "pkg:deb/ubuntu/python-bleach?arch=src?distro=esm-apps/bionic"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						}
					]
				}
			],
			"versions": [
				"2.0-1",
				"2.1.2-1"
			],
			"ecosystem_specific": {
				"ubuntu_priority": "medium"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2018/UBUNTU-CVE-2018-7753.json"
			}
		},
		{
			"package": {
				"ecosystem": "Ubuntu:20.04:LTS",
				"name": "python-bleach",
				"purl": "pkg:deb/ubuntu/python-bleach?arch=src?distro=focal"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.1.1-1"
						}
					]
				}
			],
			"versions": [
				"3.1.0-1",
				"3.1.0-2"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-bleach-doc",
						"binary_version": "3.1.1-1"
					},
					{
						"binary_name": "python3-bleach",
						"binary_version": "3.1.1-1"
					}
				],
				"ubuntu_priority": "medium"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2018/UBUNTU-CVE-2018-7753.json"
			}
		}
	],
	"references": [
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2018-7753"
		},
		{
			"type": "REPORT",
			"url": "https://github.com/mozilla/bleach/pull/356"
		},
		{
			"type": "REPORT",
			"url": "https://github.com/mozilla/bleach/commit/c5df5789ec3471a31311f42c2d19fc2cf21b35ef"
		},
		{
			"type": "REPORT",
			"url": "https://bugs.debian.org/892252"
		},
		{
			"type": "REPORT",
			"url": "https://github.com/mozilla/bleach/releases/tag/v2.1.3"
		},
		{
			"type": "REPORT",
			"url": "https://www.cve.org/CVERecord?id=CVE-2018-7753"
		}
	]
}
